css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
Please make a call, as I said, I don't understand how it's used.

So from what I understand, when all of the following condition met:

* If `white-space: pre-wrap` and
* if a non-breakable word and at least one trailing space fit within the line, and
* If subsequent spaces follow, and
* If those subsequent spaces do not fit within the line

You want to break between spaces, but not before/after?